SPEC-annex-B-elf-embedding.md

```diff
@@ -0,0 +1,29 @@
+# Annex B
+
+Annex B contains a method of embedding OmniBOR ID into elf files.
+
+### Embedded Artifact Dependency Graph Identifier
+If an artifact contains an embedded artifact dependency graph identifier, then implementations MUST conform to the file format recommendations specified in this document.
+If an artifact does not contain an embedded artifact dependency graph identifier, then implementations SHOULD look for that information in alternate locations. Tools may store and retrieve such information in alternate formats.
+
+#### Gitoid persistence by a Build Tool in ELF Objects/Executables
+When persisting the gitoid to an ELF object or an ELF executable, the build tool should create a section ```.note.omnibor``` and place the gitoid in the descriptor field of the Note entry. This section must be of type SHT_NOTE and must have the attribute SHF_ALLOC. Multiple Note entries must be created, one for each gitoid type when multiple gitoid types are involved. Each Note entry must contain the following fields in the same order as given below:
```

Comment:
"gitoid" ought to be "GitOID"

Reply:
Could you say more about the reasoning?

Reply:
Ah, I believe that's how we'd capitalized it in other places on the site and in the documentation. I haven't double-checked myself, so I could be wrong. I know at least in the Rust implementation that's how we've named it.